backends/cadence/aot/ops_registrations.py

```diff
@@ -31,7 +31,6 @@
 lib.define(
     "quantized_layer_norm(Tensor X, Tensor X_scale, Tensor X_zero_point, int[] normalized_shape, Tensor weight, Tensor bias, float eps, float output_scale, int output_zero_point) -> (Tensor Y)"
 )
-
```

Comment:
white lines removed by accident?

Reply:
no I made all cases look the same (grouping default and out overloads)